cnf-certification-test/observability/suite.go

```diff
@@ -64,7 +64,6 @@ var _ = ginkgo.Describe(common.ObservabilityTestKey, func() {
 
 	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestPodDisruptionBudgetIdentifier)
 	ginkgo.It(testID, ginkgo.Label(tags...), func() {
-		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.PodDisruptionBudgets)
```

Comment:
add skipping if there is no deployments or statefulsets :
testhelper.SkipIfEmptyAll(ginkgo.Skip, env.Deployments, env.StatefulSets)

Reply:
Done.